dpc/src/record/payload.rs

```diff
@@ -107,7 +106,7 @@ impl<'de, N: Network> Deserialize<'de> for Payload<N> {
 
 impl<N: Network> Default for Payload<N> {
     fn default() -> Self {
-        Self::from(&[])
+        Self::from(vec![])
```

Comment:
```suggestion
        Self::from(vec![0u8; N::RECORD_CIPHERTEXT_SIZE_IN_BYTES])
```

Reply:
this change appears to break some tests; I can either revert it to the current status quo, or adjust the tests - your call @howardwu.